src/openrct2-ui/interface/Window.cpp

```diff
@@ -587,7 +587,7 @@ void window_draw_widgets(rct_window* w, rct_drawpixelinfo* dpi)
 
     if ((w->flags & WF_TRANSPARENT) && !(w->flags & WF_NO_BACKGROUND))
         gfx_filter_rect(
-            dpi, w->windowPos.x, w->windowPos.y, w->windowPos.x + w->width - 1, w->windowPos.y + w->height - 1, PALETTE_51);
+            dpi, { w->windowPos, w->windowPos + ScreenCoordsXY{ w->width - 1, w->windowPos.y + w->height - 1 } }, PALETTE_51);
```

Comment:
There's an extra window pos on y arg here

Reply:
Nice! Looks like that's the issue indeed.

Reply:
Issue should be fixed. Deleted the extra windowPos and tested locally. I'll make another pull request.

Reply:
I already filed #12554, as this is a game breaker.